lib/cocoapods/command/lib.rb

```diff
@@ -123,6 +124,7 @@ def initialize(argv)
           @quick           = argv.flag?('quick')
           @allow_warnings  = argv.flag?('allow-warnings')
           @clean           = argv.flag?('clean', true)
+          @fail_fast       = argv.flag?('fail-fast', false)
```

Comment:
Would it make sense for this to become the default, and `--no-fail-fast` to disable? /c @orta